### ISSUE TITLE
How long for  training?

### ISSUE THREAD

**Issue by qinhui99** [BeloborodovDS/MobilenetSSDFace#2]
Thanks for your good job.   I used the script ' train_full' to train the model. After 12 hours later, the mbox_loss is about 1.63302.  I don't know when I should stop the training.  Can you tell me the mbox_loss when you stop the training ?

**Comment by BeloborodovDS** [comment]
@qinhui99 
For my best runs loss was actually higher, it oscillated around 2.5. However loss is not very informative, since it may change with setting of different hyperparameters. You should definitely use validation set to measure some informative performance metric like mAP (ideally this one should be > 0.85)

I was training this detector in Colaboratory, and I had to take two 12-hour sessions to train each model (almost 10000 iterations total), I saved a snapshot every ~500 iterations. I then used a script to measure mAP on all snapshots and plot progress.

By the way, in Caffe you can include testing on a validation set in your training loop, see docs for .prototxt files
